### PR TITLE
Remove patch for pblat

### PR DESCRIPTION
**Pull request by icebert**
Remove patch for pblat

### CODE REVIEW

**Comment by icebert** [comment]
@BiocondaBot please add label

File: recipes/pblat/meta.yaml [hunk 1]
```diff
@@ -7,9 +7,6 @@ package:
 source:
   url: https://github.com/icebert/pblat/archive/{{ version }}.tar.gz
   sha256: 281440d93c4a555aceb8d5dd97573f69f001af43a33a44fa4663742a04adecc3
-  #https://github.com/icebert/pblat/pull/14
-  patches:
-    - patch
 
 build:
   number: 0
```
Comment (npavlovikj) [inline recipes/pblat/meta.yaml:12]: Please increase the build number to 1.

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

File: recipes/pblat/meta.yaml (hunk 1 above)
Comment (npavlovikj) [reply recipes/pblat/meta.yaml:12]: Actually, sorry about that - I just saw that the 2.5 version hasn't been merged yet.

**Pull request closed without merging**